src/aleph/model/hashes.py

```diff
@@ -31,19 +35,19 @@
 #                                         upsert=True)
 
 
-async def get_value(key: str):
+async def get_value(key: str) -> Optional[bytes]:
     from aleph.model import fs
     if fs is None:
         raise ValueError("MongoDB fs not initialized")
 
     try:
         gridout = await fs.open_download_stream_by_name(key)
-        return await gridout.read()
+        return await gridout.reaopen_download_stream_by_named()
```

Comment:
I cannot find the definition of `reaopen_download_stream_by_named` in GridOut.
Where is it defined ?

Reply:
Typo, the "a" should not be there.